webpack/ForemanInventoryUpload/Components/PageHeader/components/DocsButton/DocsButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Button, Icon } from 'patternfly-react';
+import {
+  INVENTORY_DOCS_LINK,
```

Comment:
Can we consume the URL through helper?
I want a helper that will use documentation controller in Foreman for latest core version. In previous versions, we can still use direct link.

Reply:
That's a great idea, changed to consume it from an helper